Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt

```diff
@@ -59,8 +60,8 @@ class NavActivity : AppCompatActivity() {
                 ) {
                     val scaffoldState = rememberScaffoldState()
 
-                    val openDrawerEvent = viewModel.drawerShouldBeOpened.observeAsState()
-                    if (openDrawerEvent.value == true) {
+                    val openDrawerEvent = viewModel.drawerShouldBeOpened.collectAsState()
```

Comment:
```suggestion
                    val drawerOpen by viewModel.drawerShouldBeOpened.collectAsState()
```

Reply:
There was a reason why I didn't do `by` :shrug:

Reply:
Not committing this because it's not complete